tls/s2n_connection.h

```diff
@@ -133,6 +138,10 @@ struct s2n_connection {
     /* Indicates whether the connection should request OCSP stapling from the peer */
     unsigned request_ocsp_status : 1;
 
+    /* Indicates that the connection was created from deserialization
+     * and therefore some fields may not be set. */
```

Comment:
It might be helpful to clarify that "some fields" pretty much means that we have no / extremely limited knowledge of the original handshake.